src/Chainweb/Chainweb.hs

```diff
@@ -447,36 +448,37 @@ withChainwebInternal conf logger peer serviceSock rocksDb pactDbDir resetDb inne
             synchronizePactDb cs mCutDb
             logg Info "finished synchronizing Pact DBs"
 
-            withPactData cs cuts $ \pactData -> do
-                logg Info "start initializing miner resources"
-
-                withMiningCoordination mLogger mConf mCutDb $ \mc ->
-
-                    -- Miner resources are used by the test-miner when in-node
-                    -- mining is configured or by the mempool noop-miner (which
-                    -- keeps the mempool updated) in production setups.
-                    --
-                    withMinerResources mLogger (_miningInNode mConf) cs mCutDb mc $ \m -> do
-                        logg Info "finished initializing miner resources"
-                        let !haddr = _peerConfigAddr $ _p2pConfigPeer $ _configP2p conf
-                        inner Chainweb
-                            { _chainwebHostAddress = haddr
-                            , _chainwebChains = cs
-                            , _chainwebCutResources = cuts
-                            , _chainwebMiner = m
-                            , _chainwebCoordinator = mc
-                            , _chainwebLogger = logger
-                            , _chainwebPeer = peer
-                            , _chainwebPayloadDb = view cutDbPayloadCas $ _cutResCutDb cuts
-                            , _chainwebManager = mgr
-                            , _chainwebPactData = pactData
-                            , _chainwebThrottler = throttler
-                            , _chainwebMiningThrottler = miningThrottler
-                            , _chainwebPutPeerThrottler = putPeerThrottler
-                            , _chainwebLocalThrottler = localThrottler
-                            , _chainwebConfig = conf
-                            , _chainwebServiceSocket = serviceSock
-                            }
+            unless (_configOnlySyncPact conf) $
```

Comment:
Going forward, please don't re-indent so much code to add a single line, it makes the diff hard to review.

Reply:
In this case, what would have been the alternative to reindenting?